doc/switchRelays.md

```diff
@@ -82,3 +82,5 @@ and `value` outputs what was read.
 Its type is `uint8_t`.
 
 Session must be closed when no longer needed.
+
+The C/C++ developer needs to code the bits of `controlRelay` and decode the bits of `debugControlRelays` in his side.
```

Comment:
The FPGA code will decode the `controlRelay` integer value from C/C++ interface, override the previous input (if any), and transmit to the hardware directly.
This implies the C/C++ developer needs to track the previous input actively.
The  `debugControlRelays` can be used to check the current value in FPGA for the debugging purpose.

Reply:
Done. Thanks for improving the description.